guides/unmanaged_devices.md

```diff
@@ -0,0 +1,266 @@
+#### Keep the device physically secure
+Unmanaged devices are generally portable and often highly valuable, so you must take appropriate steps to keep the
+device physically secure.
+
+In particular, you should:
+ - keep the device hidden whenever it is being transported or stored
+ - ensure the device is stored in a secure, locked location
+ - ensure the device is adequately protected from damage at all times
+
+Cases, bags or other containers can be provided by your line manager.
+
+#### Use the device safely
+Unmanaged devices often have different physical characteristics than standard-issue equipment.
+
+You must make sure that you complete an assessment of how to use the device in a way that does not cause strain or
+long-term injury.
+
+Some specific areas to look out for are:
+ - screen size, graphics resolution and text size may not be optimal in the default configuration
+ - on-screen or built-in keyboards, touchpads or trackpads are often not suitable for long periods of use
+ - default operating system or software configuration can often not be suitable for long periods of use
+ - devices may not have the necessary connectivity options for external equipment
+
+Additional equipment or guidance can be provided by your line manager.
+
+#### Backup and secure data
+Using unmanaged devices often requires the creation of data outside of our secure, corporate repositories.
+
+You must ensure that any unique content you create is backed up to alternative persistent storage in accordance with
+the information management procedures that apply.
+
+Additional equipment such as encrypted hard drives or online services such as OneDrive can be provided by your
+line manager.
+
+#### Returning devices
+When you return your device, you must ensure that it has been treated to remove all official information and is
+ready to be used by someone else.
+
+This usually just means performing a factory reset, although in some cases additional steps may be required.
+You must ensure that there are no online accounts linked to the device, such as Microsoft or Apple IDs.
+
+### Controls for Windows laptops
+
+#### Don't set a BIOS password
+BIOS passwords don't add any greater security to the device and primarily only help to prevent accidental
+mis-configuration.
+
+Setting such passwords can make it more difficult to reuse the device so do not set a BIOS password.
+
+#### Use the latest stable desktop operating system
+The device is likely to come with a suitable version of Windows 10 pre-installed, but if not then you should ensure
+that Windows 10 Professional or Enterprise is installed on it, but do not remove any existing recovery partition.
+
+Guidance on this can be provided by your line manager.
+
+You must ensure that the device has the latest stable version.
+
+Do not use server versions or different operating systems, e.g. any Linux distribution.
+
+If you need to work with a different operating system then you must use a virtualisation technology.
```

Comment:
Maybe suggest WSL as a sensible (the best ;-p) option?

https://docs.microsoft.com/en-us/windows/wsl/about

Reply:
I'll add a mention of it.